File: projeto/UI/pinguino/src/app/models/estado.ts
import { Pais } from "./pais";

export class Estado {

    constructor() { 
        this.id = 0;
        this.pais = new Pais();
        this.descricao = '';
        this.sigla = '';
        this.ativo = true;
    }

    id: number;
    pais: Pais;
    descricao: string;
    sigla: string;
    ativo: boolean;
}

File: projeto/API/PinguinoApp/PinguinoApp.API/Models/Estado.cs
namespace PinguinoApp.API.Models
{
    public class Estado
    {
        public int Id { get; set; }

        public string Descricao { get; set; }

        public string Sigla { get; set; }

        public Pais Pais { get; set; }

        public bool Ativo { get; set; }
    }
}
